src/set_theory/ordinal/fixed_point.lean

```diff
@@ -1,7 +1,7 @@
 /-
 Copyright (c) 2022 Violeta Hernández Palacios. All rights reserved.
```

Comment:
This should be updated as well.

Reply:
Isn't it the case that the first line of the copyright person should only credit one person?

Reply:
No. Bhavik and I pyright a lot of files.